docs/source/linchpincli_linchpin_invgen.rst

```diff
@@ -1,24 +0,0 @@
-Invgen
```

Comment:
For this and other removed commands (config, invgen, layout, topologies, validate), commenting it out causes more problems than it solves. One issue is that sphinx will warn about these documents not being referenced by a toctree. For example:
`checking consistency... /home/semyers/ve/cinch/linchpin/docs/source/linchpincli_linchpin_config.rst:: WARNING: document isn't included in any toctree`

Another is that the files as-written now will need to be rewritten to match the formatting I've used in the docs for init, up, destroy, rise, and drop.

If the warnings don't bother you, I can update the formatting to conform to the updates I've made and leave them in-place as stubs. Just let me know what you'd prefer.